kitsune/sumo/static/sumo/js/surveygizmo.js

```diff
@@ -1,23 +1,23 @@
-;(function() {
+/* globals $:false */
+(function() {
     function launchWindow(url) {
-        var sg_div = document.createElement("div");
+        var sg_div = document.createElement('div');
         sg_div.innerHTML = (
             '<h1>You have been selected for a survey</h1>' +
             '<p>We appreciate your feedback!</p><p><a href="' + url + '">Please click here to start it now.</a></p>' +
             '<a href="#" onclick="document.getElementById(\'sg-popup\').style.display = \'none\';return false;">No, thank you.</a>'
         );
-        sg_div.id = "sg-popup";
+        sg_div.id = 'sg-popup';
         document.body.appendChild(sg_div);
     }
 
     var surveys = {
         site_survey: function() {
-            var surveyGizmoURL = $('body').data('surveygizmo-url') ||
-                "https://www.surveygizmo.com/s3/popup/1002970/46488f9ad4eb";
-            // This is adapted from a snipptet given by Survey Gizmo.
+            // This is adapted from a snippet given by Survey Gizmo.
             // It has been formatted to fit your screen. Bug 782809.
-            if (!$.cookie('hasSurveyed')) {
-                window.addEventListener("load", function(e) {
+            var surveyGizmoURL = $('body').data('surveygizmo-url');
+            if (surveyGizmoURL && !$.cookie('hasSurveyed')) {
```

Comment:
It is a bit hard to tell from the diff, but here I made it so there is no default survey. The former default survey is being replaced with the Beacon.

Most of the other changes in this function are simply style changes.